### ISSUE TITLE
Bowling Green, OH is missing. 

### ISSUE THREAD

**Issue by williamspatrick** [acrylic-origami/the-other-one#1]
I was confused by Bowling Green, OH not being the real one (since it is the one with the university I’ve at least heard of) and stumbled upon your site. The Ohio one is missing from your database, which makes KY seem much more important than it is.

**Comment by acrylic-origami** [comment]
You are absolutely correct, there was a bug in my data sanitization/loading. Seems like alternate names don't always include the most common name as I had assumed. Fixing that now. Thanks a bunch for reporting this!
